### PR TITLE
Fix: mock a11y api request for court events glossary

### PR DESCRIPTION
**Pull request by commitBlob**
### Change description ###
Mock courteventglossary API call which causes a11y tests to fail

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by ChrisS1512 — DISMISSED** [review]

File: src/test/a11y/a11y.ts [hunk 1]
```diff
@@ -8,27 +8,33 @@ import fs from 'fs';
...
 const courtData = allCourtData[0];
 const liveCaseData = JSON.parse(rawDataLive).results;
+const statusDescriptionData = JSON.parse(rawDataStatusDescription);
 
 sinon.stub(CourtRequests.prototype, 'getCourt').returns(courtData);
 sinon.stub(CourtRequests.prototype, 'getCourtByName').returns(courtData);
 sinon.stub(CourtRequests.prototype, 'getFilteredCourts').returns(allCourtData);
 sinon.stub(CourtRequests.prototype, 'getAllCourts').returns(allCourtData);
 sinon.stub(LiveCaseRequests.prototype, 'getLiveCases').returns(liveCaseData);
+sinon.stub(dataManagementApi, 'get').withArgs('/courteventglossary').resolves({data: statusDescriptionData});
```
Comment (kurt-uk) [inline src/test/a11y/a11y.ts:37]: match the other mocks and just stick with mocking the request service

**Review by kurt-uk — CHANGES_REQUESTED** [review]

File: src/main/service/statusDescriptionService.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
-import {SearchDescriptionRequests} from '../resources/requests/searchDescriptionRequests';
+import {StatusDescriptionRequests} from '../resources/requests/statusDescriptionRequests';
 import {CourtService} from '../service/courtService';
 
-const searchDescriptionRequests = new SearchDescriptionRequests();
+const searchDescriptionRequests = new StatusDescriptionRequests();
```
Comment (kurt-uk) [inline src/main/service/statusDescriptionService.ts:4]: i know this isnt your work and youve refactored but can you change the variable to match?

File: src/test/unit/requests/searchDescriptionRequests.test.ts [hunk 3]
```diff
@@ -1,10 +1,10 @@
 import sinon from 'sinon';
 import {dataManagementApi} from '../../../main/resources/requests/utils/axiosConfig';
-import {SearchDescriptionRequests} from '../../../main/resources/requests/searchDescriptionRequests';
+import {StatusDescriptionRequests} from '../../../main/resources/requests/statusDescriptionRequests';
 import fs from 'fs';
 import path from 'path';
 
-const searchDescriptionRequests = new SearchDescriptionRequests();
+const searchDescriptionRequests = new StatusDescriptionRequests();
```
Comment (kurt-uk) [inline src/test/unit/requests/searchDescriptionRequests.test.ts:7]:
```suggestion
const statusDescriptionRequests = new StatusDescriptionRequests();
```

File: src/test/unit/service/statusDescriptionService.test.ts [hunk 4]
```diff
@@ -3,11 +3,11 @@ import sinon from 'sinon';
 import { expect } from 'chai';
 import fs from 'fs';
 import path from 'path';
-import {SearchDescriptionRequests} from '../../../main/resources/requests/searchDescriptionRequests';
+import {StatusDescriptionRequests} from '../../../main/resources/requests/statusDescriptionRequests';
 
 const statusDescriptionService = new StatusDescriptionService();
 
-const searchDescriptionRequests = SearchDescriptionRequests.prototype;
+const searchDescriptionRequests = StatusDescriptionRequests.prototype;
```
Comment (kurt-uk) [inline src/test/unit/service/statusDescriptionService.test.ts:10]:
```suggestion
const statusDescriptionRequests = StatusDescriptionRequests.prototype;
```

**Review by kurt-uk — CHANGES_REQUESTED** [review]

**Review by kurt-uk — APPROVED** [review]

**Review by johnoneill46 — APPROVED** [review]

**Pull request merged**